test/unit/test_devices/test_eos_device.py

```diff
@@ -50,7 +50,7 @@ def test_bad_config_list(self):
         commands = ["interface Eth1", "apons"]
 
         with self.assertRaisesRegex(CommandListError, commands[1]):
```

Comment:
No need for this test case since the config_list method now just calls the config method.